templates/zerver/help/add-custom-emoji.md

```diff
@@ -3,8 +3,10 @@
 If enabled by your Zulip organization administrator, you can add custom
 emojis to your organization for other members to use.
 
-1. Go to the [Emoji Settings](/#organization/emoji-settings)
-{!admin.md!}
+{setting|Custom emoji}
```

Comment:
@timabbott I used the macro syntax {macroname|arg1|arg2} in the API docs for consistency's sake, but then it occurred to me that I might want to be a bit more normal and do something like !macroname(arg1, arg2) as per [this](https://github.com/zulip/zulip/pull/5662/files), and the API docs can be abnormal because they have so many references to more normal functions using normal function syntax. What do you think?

Reply:
I don't have strong feelings; I think the pipe syntax might make parsing a bit simpler, since it's clear whether or not you want a leading space.  The one thing that I might consider changing is that I'd probably do a machine-readable style identifier like `{setting|emoji-settings}`  for this sort of syntax, since then one doesn't need to think about case.  Ideally, the machine-readable identifers should match the thing that appears in the URL for that settings page.

Reply:
agreed. 
Another small thing is I think we want `settings_tab` here instead of `setting`, since I think of setting as corresponding to e.g. an individual checkbox.